lib/horoshop/connection.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'faraday'
+class Horoshop
+  # Module for check connection
+  module Connection
+    ERROR = { status: 'HTTP_ERROR', message: 'UNKNOWN SERVER ERROR' }.freeze
+
+    def post(horoshop:, url:, body:)
+      connection(horoshop).post(url, body)
+    rescue Faraday::Error => e
+      error_status = e.response[:status]
+      error_body = e.response&.dig(:body)
```

Comment:
why you using safe navigation before `dig` in this row but do not doe this in row 12

Reply:
Because we discussed that the body can come empty, but the status is always there